pkg/controller/volume/attachdetach/cache/desired_state_of_world.go

```diff
@@ -158,6 +158,9 @@ type nodeManaged struct {
 
 // The volume object represents a volume that should be attached to a node.
 type volumeToAttach struct {
+	// allowMultiAttach shows whether the volume is allowed to multi-attach
+	allowMultiAttach bool
```

Comment:
Please rename `multiAttachErrorReported` and update comment similar to my other suggestion.